2022-08-01 15:54:58,752 INFO    Thread-13 :8718 [wandb_setup.py:_flush():76] Configure stats pid to 8718
2022-08-01 15:54:58,752 INFO    Thread-13 :8718 [wandb_setup.py:_flush():76] Loading settings from /home/kana/.config/wandb/settings
2022-08-01 15:54:58,752 INFO    Thread-13 :8718 [wandb_setup.py:_flush():76] Loading settings from wandb/settings
2022-08-01 15:54:58,752 INFO    Thread-13 :8718 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'noisy_label_cleaning_a', 'entity': 'kka-na', 'root_dir': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script', 'run_id': '218auubg', 'sweep_param_path': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/sweep-838wzr8r/config-218auubg.yaml', 'sweep_id': '838wzr8r'}
2022-08-01 15:54:58,752 WARNING Thread-13 :8718 [wandb_setup.py:_flush():76] Could not save program above cwd: ../src/hyp_tuning.py
2022-08-01 15:54:58,752 INFO    Thread-13 :8718 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program': '../src/hyp_tuning.py'}
2022-08-01 15:54:58,752 INFO    Thread-13 :8718 [wandb_init.py:_log_setup():461] Logging user logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_155458-218auubg/logs/debug.log
2022-08-01 15:54:58,752 INFO    Thread-13 :8718 [wandb_init.py:_log_setup():462] Logging internal logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_155458-218auubg/logs/debug-internal.log
2022-08-01 15:54:58,756 INFO    Thread-13 :8718 [wandb_init.py:init():495] calling init triggers
2022-08-01 15:54:58,756 INFO    Thread-13 :8718 [wandb_init.py:init():498] wandb.init called with sweep_config: {'BATCH_SIZE_PER_IMAGE': 256, 'IMS_PER_BATCH': 2, 'iteration': 300, 'learning_rate': 0.00033022349858873013, 'model': None}
config: {}
2022-08-01 15:54:58,760 INFO    Thread-13 :8718 [wandb_init.py:init():548] starting backend
2022-08-01 15:54:58,760 INFO    Thread-13 :8718 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-01 15:54:58,770 INFO    Thread-13 :8718 [backend.py:ensure_launched():217] starting backend process...
2022-08-01 15:54:58,775 INFO    Thread-13 :8718 [backend.py:ensure_launched():222] started backend process with pid: 8847
2022-08-01 15:54:58,776 INFO    Thread-13 :8718 [wandb_init.py:init():558] backend started and connected
2022-08-01 15:54:58,792 INFO    Thread-13 :8718 [wandb_run.py:_config_callback():1131] config_cb None None {'BATCH_SIZE_PER_IMAGE': 256, 'IMS_PER_BATCH': 2, 'iteration': 300, 'learning_rate': 0.00033022349858873013, 'model': None}
2022-08-01 15:54:58,795 INFO    Thread-13 :8718 [wandb_init.py:init():634] updated telemetry
2022-08-01 15:54:58,796 INFO    Thread-13 :8718 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2022-08-01 15:54:59,556 INFO    Thread-13 :8718 [wandb_run.py:_on_init():1919] communicating current version
2022-08-01 15:54:59,604 INFO    Thread-13 :8718 [wandb_run.py:_on_init():1923] got version response 
2022-08-01 15:54:59,605 INFO    Thread-13 :8718 [wandb_init.py:init():700] starting run threads in backend
2022-08-01 15:55:00,747 INFO    Thread-13 :8718 [wandb_run.py:_console_start():1893] atexit reg
2022-08-01 15:55:00,751 INFO    Thread-13 :8718 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2022-08-01 15:55:00,751 INFO    Thread-13 :8718 [wandb_run.py:_redirect():1771] Redirecting console.
2022-08-01 15:55:00,751 ERROR   Thread-13 :8718 [wandb_run.py:_redirect():1830] Failed to redirect.
Traceback (most recent call last):
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1823, in _redirect
    out_redir.install()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 702, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 652, in add_fd
    self._register()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 637, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-08-01 15:55:00,752 INFO    Thread-13 :8718 [wandb_init.py:init():732] run started, returning control to user process
2022-08-01 15:55:00,753 INFO    Thread-13 :8718 [wandb_run.py:_finish():1680] finishing run kka-na/noisy_label_cleaning_a/218auubg
2022-08-01 15:55:00,753 INFO    Thread-13 :8718 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 1
2022-08-01 15:55:00,754 INFO    Thread-13 :8718 [wandb_run.py:_restore():1834] restore
2022-08-01 15:55:01,056 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 527
}

2022-08-01 15:55:01,443 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-01 15:55:01,579 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5911
}

2022-08-01 15:55:01,681 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5911
}

2022-08-01 15:55:01,784 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5911
}

2022-08-01 15:55:01,886 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5911
}

2022-08-01 15:55:01,988 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3644
  total_bytes: 5911
}

2022-08-01 15:55:02,091 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3644
  total_bytes: 5911
}

2022-08-01 15:55:02,194 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3644
  total_bytes: 5911
}

2022-08-01 15:55:02,296 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5911
  total_bytes: 5911
}

2022-08-01 15:55:02,399 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5911
  total_bytes: 5911
}

2022-08-01 15:55:02,502 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5911
  total_bytes: 5911
}

2022-08-01 15:55:02,604 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5911
  total_bytes: 5911
}

2022-08-01 15:55:02,707 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5911
  total_bytes: 5911
}

2022-08-01 15:55:02,810 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5911
  total_bytes: 5911
}

2022-08-01 15:55:03,033 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5911
  total_bytes: 5911
}

2022-08-01 15:55:03,561 INFO    Thread-13 :8718 [wandb_run.py:_on_finish():1991] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5911
  total_bytes: 5911
}
local_info {
}

